packages/react-notifications/src/components/notification-kinds/api-error-message/api-error-message.spec.tsx

```diff
@@ -1,17 +1,18 @@
+import { mocked } from 'ts-jest/utils';
 import React from 'react';
 import { render, wait } from '@testing-library/react';
 import { IntlProvider } from 'react-intl';
 import { reportErrorToSentry } from '@commercetools-frontend/sentry';
-import { ApiErrorMessage } from './api-error-message';
+import ApiErrorMessage from './api-error-message';
 
 jest.mock('@commercetools-frontend/sentry');
 
-const renderMessage = ui =>
+const renderMessage = (ui: React.ReactElement) =>
   render(<IntlProvider locale="en">{ui}</IntlProvider>);
 
 describe('render', () => {
   it('should show message for InvalidInput', () => {
-    const error = { code: 'InvalidInput' };
+    const error = { code: 'InvalidInput', message: 'message-content' };
```

Comment:
Besides the missing "required" fields, I didn't need to change a single test. That was very nice!